src/components/search-bar/story-header.dev.inc.tsx

```diff
@@ -94,18 +97,39 @@ const StoryHeader: React.FC<StoryHeaderProps> = ({ children }) => {
                 </li>
               </ul>
             </div>
-            <div className="header__menu-profile header__button">
-              <a href="/" className="hide-linkstyle">
+            {userProfile || (
+              <div className="header__menu-profile header__button">
                 <img src={profileIcon} alt="Profile" />
```

Comment:
Do we not want to show the newly added user name to the story?

Reply:
We want to show it if `userProfile` is defined, which it is in the "header -> user menu" story. But we don't want that in the "header -> search" story. :))